connector_jira_tempo_base/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'JIRA Connector Tempo (base)',
+    'version': '12.0.1.0.0',
+    'category': 'Connector',
+    'website': 'https://github.com/OCA/connector-jira',
+    'author':
+        'Brainbean Apps, '
```

Comment:
```suggestion
        'Brainbean Apps, '
        'Camptocamp, '
```

Since it's extracted from `connector_jira_tempo`

Reply:
Done